terraform/modules/cognito/outputs.tf

```diff
@@ -1,9 +1,11 @@
 output outputs {
   value = {
-    name             = aws_cognito_user_pool_client.app_client.name
-    app_client       = aws_cognito_user_pool_client.app_client
-    user_pool_domain = "https://${aws_cognito_user_pool_domain.main.domain}.auth.${local.region}.amazoncognito.com"
-    user_pool_id     = aws_cognito_user_pool.concourse.id
-    issuer           = "https://cognito-idp.${local.region}.amazonaws.com/${aws_cognito_user_pool.concourse.id}"
+    name                = aws_cognito_user_pool_client.app_client.name
+    app_client          = aws_cognito_user_pool_client.app_client
+    user_pool_domain    = "https://${aws_cognito_user_pool_domain.main.domain}.auth.${local.region}.amazoncognito.com"
+    user_pool_id        = aws_cognito_user_pool.concourse.id
+    user_pool_arn       = aws_cognito_user_pool.concourse.arn
+    user_pool_client_id = aws_cognito_user_pool_client.app_client.id
```

Comment:
You don't need this one. Consumers can just look at the `app_client` output above.

Similarly, if you create a sinlge `user_pool` output, then consumers can grab the `id`, `arn`, and `client_id` out of them.

Reply:
Deja vu.